client/containers/Users/Dialogs/RemoveMultiFactorDialog.jsx

```diff
@@ -47,6 +55,21 @@ export default connectContainer(class extends Component {
     const message = getDialogMessage(messageFormat, 'username',
       getName(user, userFields, languageDictionary));
 
+    const fields = _.cloneDeep(userFields) || [];
+    const providers = user && user.multifactor ? user.multifactor : [];
+    useMfaField(true, fields, providers);
+
+    const allowedFields = [ 'user_id', 'multifactor' ];
+    const filteredFields = _.filter(fields,
+      field => _.includes(allowedFields, field.property));
+
+    const UserFieldsFormInstance = UserFieldsForm('remove-mfa', this.onSubmit.bind(this));
```

Comment:
Why is this instantiated here? seems like it might be the wrong place.

Reply:
We could create a constructor and add it there. However, I was trying to be consistent with all the other forms: https://github.com/auth0-extensions/auth0-delegated-administration-extension/blob/e2751f438adbf492d1e00dc17fff350a031cca28/client/containers/Users/Dialogs/EmailChangeDialog.jsx#L70

Reply:
Ah, ok. didn't realize you had copied from other forms. that might make some of my other points moot as well.
consistency FTW!